src/components/overal-statistics/PopularGameTimeChart.vue

```diff
@@ -20,8 +20,8 @@ export default class PopularGameTimeChart extends Vue {
     const gameHour = this.popularGameHour;
     if (!gameHour) return [];
 
-    const times = gameHour?.playTimePerHour.slice(4);
-    times.pop();
```

Comment:
slice(4) caused bars 00:00, 00:15, 00:30, 00:45 and pop caused 23:45 to not go through